packages/example/gatsby-config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     title: 'Gatsby Theme Carbon',
     description: 'A Gatsby theme for the carbon design system',
     keywords: 'gatsby,theme,carbon',
+    lang: 'en',
```

Comment:
I think maybe we should leave this out of the example, it doubles as the starter-kit and folks might not know if they need to leave it in.

Would could update [https://gatsby-theme-carbon.now.sh/guides/configuration#site-title-and-description](this part of the config guide) to just be Site Metadata and add a note there maybe? "**If necessary**, you can change the default language of your document using the `lang` property" (maybe dont need the emphasis haha)

Some thing like 
```suggestion
```